gno.land/pkg/gnoland/mock_test.go

```diff
@@ -53,7 +53,7 @@ func (m *mockEventSwitch) RemoveListener(listenerID string) {
 type mockVMKeeper struct {
 	addPackageFn                func(sdk.Context, vm.MsgAddPackage) error
 	callFn                      func(sdk.Context, vm.MsgCall) (string, error)
-	queryFn                     func(sdk.Context, string, string) (string, error)
+	queryFn                     func(sdk.Context, vm.EvalCfg) (string, error)
```

Comment:
MsgQuery would make more sense. 
Why Cfg?

Reply:
See this comment: https://github.com/gnolang/gno/pull/2949#discussion_r1943537363

I think I agree with them. If we keep it as MsgQuery, it makes it feel like it’s a transaction, which it isn’t.

Reply:
Then just QueryEval. I don't think it makes sense to call an object 'cfg' here, it's not a config.toml kinda thing.

Reply:
QueryEval